discovery-provider/scripts/prod-server.sh

```diff
@@ -6,5 +6,28 @@
 #   "-b :5000" accept requests on port 5000
 #   "--access-logfile - --error-logfile" - log to stdout/stderr
 #   "src.wsgi:app" - app entry point in format: $(MODULE_NAME):$(VARIABLE_NAME)
-exec gunicorn -b :5000 --access-logfile - --error-logfile - src.wsgi:app --log-level=debug --workers 2 --threads 8
 
+# Use specified number of workers if present
+if [[ -z "${audius_gunicorn_workers}" ]]
```

Comment:
is there an associated config update that is missing? https://github.com/AudiusProject/audius-protocol/blob/896eea4fa2507fbc5aa81cb87fdf47ecca793d63/discovery-provider/default_config.ini#L1

i also think these should be audius_discprov_gunicorn_workers if we follow the format for other configs

Reply:
oh because these are environment configs and not used by the service this doesn't really matter?

Reply:
yeah -- the .ini is for python, this is before python's even running

Reply:
config is all updated on the k8s side

Reply:
there's a world in which we want this locally too, but we never run gunicorn locally as it stands, so it doesn't make a diff